src/journey-maps/angular/journey-maps.interfaces.ts

```diff
@@ -54,6 +54,20 @@ export interface SbbViewportOptions {
   boundingBoxPadding?: number;
 }
 
+export interface SbbHomeButtonOptions {
```

Comment:
Should we add a comment that it is either mapcenter or bbox? Or is this clear?

Or even something like this?

    export interface SbbHomeButtonOptions {
      options: SbbMapCenterOptions | SbbBoundingBoxOptions;
    }
    
    export interface SbbMapCenterOptions {
      mapCenter: LngLatLike;
      zoomLevel: number;
    }
    
    export interface SbbBoundingBoxOptions {
      boundingBox: LngLatBoundsLike;
      padding?: number;
    }

Reply:
I already added a comment here https://github.com/sbb-design-systems/sbb-angular/pull/1483/files#diff-3f390bf92d0afb141a4112538b5440adddebb8c738ecc8e2ab5e3ad1bde639cfR316 which if ESTA works the same as our old documentation generation, should show up in the documentation section of the example app.

I was thinking the same for the Typscript Types, but then we would need to refactor ViewportOptions as well. And for that I would need to make changes to your unmerged branch ...

Reply:
@sebisteiner what about this (using `type`) ? 

```
export type SbbHomeButtonOptions = SbbMapCenterOptions | SbbBoundingBoxOptions;

export interface SbbMapCenterOptions {
  /**
   * The desired center of the map. You should pass an array with two numbers,
   * the first one being the longitude and the second one the latitude.
   */
  mapCenter?: LngLatLike;
  /** The desired zoom level of the map. */
  zoomLevel?: number;
}

export interface SbbBoundingBoxOptions {
  /** The desired bounding box of the map. */
  boundingBox: LngLatBoundsLike;
  /** The amount of padding in pixels to add to the given bounding box. */
  padding?: number;
}
```